src/components/molecules/UserStatusManager/components/UserStatusPanel/UserStatusPanel.scss

```diff
@@ -0,0 +1,35 @@
+@import "scss/constants";
+
+$picker-icon--size: 35px;
+$delete-icon--size: 25px;
+$status--spacing: 5px 0;
+$icon--spacing: 5px;
```

Comment:
```suggestion
$icon--spacing: 5px;
$status--spacing: $icon--spacing 0;
```

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/1532/commits/3d83fdf0a1c6006435e8500d063e921a7b822160